sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/ThrowReservedExceptions.cs

```diff
@@ -7,28 +7,27 @@ public class ThrowReservedExceptions
         public void Method1()
         {
             throw new Exception(); // Noncompliant {{'System.Exception' should not be thrown by user code.}}
-//                    ^^^^^^^^^
+//                ^^^^^^^^^^^^^^^
 
             throw new ApplicationException(); // Noncompliant {{'System.ApplicationException' should not be thrown by user code.}}
-//                    ^^^^^^^^^^^^^^^^^^^^
+//                ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
             throw new SystemException(); // Noncompliant {{'System.SystemException' should not be thrown by user code.}}
-//                    ^^^^^^^^^^^^^^^
+//                ^^^^^^^^^^^^^^^^^^^^^
 
             throw new ExecutionEngineException(); // Noncompliant {{'System.ExecutionEngineException' should not be thrown by user code.}}
-//                    ^^^^^^^^^^^^^^^^^^^^^^^^
+//                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
             throw new IndexOutOfRangeException(); // Noncompliant {{'System.IndexOutOfRangeException' should not be thrown by user code.}}
-//                    ^^^^^^^^^^^^^^^^^^^^^^^^
+//                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
             throw new NullReferenceException(); // Noncompliant {{'System.NullReferenceException' should not be thrown by user code.}}
-//                    ^^^^^^^^^^^^^^^^^^^^^^
+//                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
             throw new OutOfMemoryException(); // Noncompliant {{'System.OutOfMemoryException' should not be thrown by user code.}}
-//                    ^^^^^^^^^^^^^^^^^^^^
+//                ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-            var e = new OutOfMemoryException(); // Noncompliant {{'System.OutOfMemoryException' should not be thrown by user code.}}
-//                      ^^^^^^^^^^^^^^^^^^^^
+            var e = new OutOfMemoryException(); // Compliant

```

Comment:
I don't particularly like this part where we do not raise when a new instance is created... But perhaps we have another rule that raises if you create exception but do not throw it.